docs/build-notifications.rst

```diff
@@ -20,8 +20,8 @@ Take these steps to enable build notifications using email:
 
 You should now get notified by email when your builds fail!
 
-Webhooks
---------
+Build Status Webhooks
+------------------------------
```

Comment:
Heh, sorry. Not my best PR suggestion :D

Reply:
Woops :D